packages/azure-http-specs/specs/azure/resource-manager/resources/extension.tsp

```diff
@@ -0,0 +1,559 @@
+    
+    Expected response body:
+    ```json
+    {
+      "value": [{
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/extensionsResources/extension",
+        "name": "extension",
+        "type": "Azure.ResourceManager.Resources/extensionsResources",
+        "properties":{
+          "description": "valid",
+          "provisioningState": "Succeeded"
+        },
+        "systemData": {
+          "createdBy": "AzureSDK",
+          "createdByType": "User",
+          "createdAt": <any date>,
+          "lastModifiedBy": "AzureSDK",
+          "lastModifiedAt": <any date>,
+          "lastModifiedByType": "User",
+        }
+      }]
+    }
+    ```
+    
+    Resource LIST extension resources by resource group.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Resources/extensionResources
+    Expected query parameter: api-version=2023-12-01-preview
+    
+    Expected response body:
+    ```json
+    {
+      "value": [{
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Resources/extensionsResources/extension",
+        "name": "extension",
+        "type": "Azure.ResourceManager.Resources/extensionsResources",
+        "properties":{
+          "description": "valid",
+          "provisioningState": "Succeeded"
+        },
+        "systemData": {
+          "createdBy": "AzureSDK",
+          "createdByType": "User",
+          "createdAt": <any date>,
+          "lastModifiedBy": "AzureSDK",
+          "lastModifiedAt": <any date>,
+          "lastModifiedByType": "User",
+        }
+      }]
+    }
+    ```
+    
+    Resource LIST extension resources by resource.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Resources/topLevelTrackedResources/top/providers/Azure.ResourceManager.Resources/extensionResources
+    Expected query parameter: api-version=2023-12-01-preview
+    
+    Expected response body:
+    ```json
+    {
+      "value": [{
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Resources/topLevelTrackedResources/top/providers/Azure.ResourceManager.Resources/extensionsResources/extension",
+        "name": "extension",
+        "type": "Azure.ResourceManager.Resources/extensionsResources",
+        "properties":{
+          "description": "valid",
+          "provisioningState": "Succeeded"
+        },
+        "systemData": {
+          "createdBy": "AzureSDK",
+          "createdByType": "User",
+          "createdAt": <any date>,
+          "lastModifiedBy": "AzureSDK",
+          "lastModifiedAt": <any date>,
+          "lastModifiedByType": "User",
+        }
+      }]
+    }
+    ```
+    """)
+  listByParent is ArmResourceListByParent<ExtensionsResource>;
```

Comment:
Sorry, one last comment, `listByScope`, and regen summary.

Reply:
Fixed by [7a391bc](https://github.com/Azure/typespec-azure/pull/1924/commits/7a391bc5dbf9585a9615e1346cb63bc2a7b7b5d9)